libs/portals/admin/regulations-admin/src/components/EditBasics.tsx

```diff
@@ -234,6 +237,38 @@ export const EditBasics = () => {
                   readOnly
                 />
               </Box>
+            ) : (
+              ministries.length > 0 &&
+              !draft.signatureText.value && (
+                <Select
+                  size="sm"
+                  label={t(m.regulationAdminMinistries)}
+                  name="setMinistry"
+                  isSearchable
+                  value={
+                    draft.ministry.value
+                      ? {
+                          value: draft.ministry.value,
+                          label: draft.ministry.value,
+                        }
+                      : undefined
+                  }
+                  placeholder={t(editorMsgs.selectMinistry)}
+                  options={[
+                    {
+                      label: t(editorMsgs.selectMinistry),
+                      value: undefined,
+                    },
+                    ...ministries.map((ministry) => ({
+                      value: ministry.name,
+                      label: ministry.name,
+                    })),
+                  ]}
+                  required={false}
+                  onChange={(option) => actions.setMinistry(option?.value)}
```

Comment:
If option is falsy, you set the ministry to null or undefined, I assume. Is this intended?

Reply:
yess it is, it should not be required to select a ministry at this stage. It's optional.